pg-auto-failover.spec

```diff
@@ -9,11 +9,11 @@ Summary:	Postgres extension for automated failover and high-availability
 Name:		%{sname}%{?pkginfix}_%{pgmajorversion}
 Provides:	%{sname}_%{pgmajorversion}
 Conflicts:	%{sname}_%{pgmajorversion}
-Version:	1.2
+Version:	1.3.1.citus
```

Comment:
Is there a reason to suffix with Citus here?

Reply:
Yes, the reason is to easily differentiate our packages from ones installed through other means, i.e. pgdg or OS packages.